backend/src/services/stripe/stripe.service.ts

```diff
@@ -0,0 +1,67 @@
+import StripeApi from 'stripe';
+
+import {
+  PaymentCurrency,
+  PaymentMethod,
+  PaymentMode,
+} from '~/common/enums/enums';
+
+const QUANTITY_OF_ITEMS = 1;
+const OPERATION_NAME = 'Replenish your funds on GuruHub';
+const CENTS_IN_ONE_DOLLAR = 100;
+const API_VERSION = '2022-08-01';
```

Comment:
And maybe move this value to env or somewhere else, doesn't seem right to have it here